identify/extensions.py

```diff
@@ -194,6 +194,7 @@
     'php4': {'text', 'php'},
     'php5': {'text', 'php'},
     'phtml': {'text', 'php'},
+    'piskel': {'text', 'image', 'piskel', 'json'},
```

Comment:
the `image` tag was an idea that didn't really pan out so I'm just going to remove it here

Reply:
That's a shame, because in this case the image tag is just what we want!